### ISSUE TITLE
Exception in thread "main" java.lang.NoSuchMethodError

### ISSUE THREAD

**Issue by breandan** [kotest/kotest#5548]
Running a test via the sidebar icon results in the following error:

```
Exception in thread "main" java.lang.NoSuchMethodError: io.kotlintest.Description$Companion.spec(Lkotlin/reflect/KClass;)Lio/kotlintest/Description;
	at io.kotlintest.runner.console.SpecAwareTestFilter.<init>(SpecAwareTestFilter.kt:32)
	at io.kotlintest.runner.console.KotlinTestConsoleRunner.execute(KotlinTestConsoleRunner.kt:12)
	at io.kotlintest.runner.console.LauncherKt.main(launcher.kt:17)
```

KotlinTest plugin version: 3.3.0.9
IntelliJ IDEA version:

```
IntelliJ IDEA 2019.1 EAP (Ultimate Edition)
Build #IU-191.5849.21, built on February 27, 2019
IntelliJ IDEA EAP User
Expiration date: March 29, 2019
JRE: 1.8.0_202-release-1483-b31 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-45-generic
```

**Comment by sksamuel** [comment]
What version of KotlinTest ?

**Comment by breandan** [comment]
3.2.1

**Comment by sksamuel** [comment]
You need the latest 3.3.0 for the intellij plugin to work.
https://oss.sonatype.org/content/repositories/snapshots/io/kotlintest/kotlintest-runner-jvm/
You'll need that repo adding to your build as well.

This is just while the plugin is in beta.

**Comment by sksamuel** [comment]
3.3.0.10 is now released compiled against KotlinTest 3.3.0
This is the first "release" version of both.
Please make sure you have upgraded both and you should be good to go!
(It might take a day before Jetbrains approve the plugin release)
